python_modules/dagster/dagster_tests/cli_tests/command_tests/file_with_local_import.py

```diff
@@ -1,7 +1,18 @@
 # type: ignore
 import dummy_local_file  # noqa: F401
+from dagster import job, op
 
-from dagster_tests.cli_tests.command_tests.test_cli_commands import (
```

Comment:
this (and a couple of other similar fixes below) are no longer needed anymore now that we are installing in legacy editable mode, but can't hurt to keep either